test/NuGet.Core.Tests/NuGet.Commands.Test/PackageSearchRunnerTests.cs

```diff
@@ -0,0 +1,280 @@
+                    }}
+                    ],
+
+                    ""@context"":
+                    {{
+                        ""@vocab"": ""http://schema.nuget.org/services#"",
+                        ""comment"": ""http://www.w3.org/2000/01/rdf-schema#comment""
+                    }}
+                }}";
+
+            const string exactMatchGetMetadataResult = @"{
+                            ""@type"": ""Package"",
+                            ""count"": 1,
+                            ""items"": [
+                                {
+                                    ""@type"": ""catalog:CatalogPage"",
+                                    ""count"": 1,
+                                    ""items"": [
+                                        {
+                                            ""@type"": ""Package"",
+                                            ""catalogEntry"": {
+                                                ""@type"": ""PackageDetails"",
+                                                ""authors"": ""James Newton-King"",
+                                                ""id"": ""Fake.Newtonsoft.Json"",
+                                                ""version"": ""12.0.3"",
+                                                ""totalDownloads"": 531607259
+                                            },
+                                        }
+                                    ],
+                                    ""lower"": ""1.0.0"",
+                                    ""upper"": ""13.0.0""
+                                }
+                            ],
+                            ""@context"": {
+                                ""@vocab"": ""http://schema.nuget.org/schema#"",
+                                ""catalog"": ""http://schema.nuget.org/catalog#""
+                            }
+                        }";
+
+            mockServer.Get.Add("/v3/index.json", r => index);
+            mockServer.Get.Add($"/v3/registration5-semver1/fake.newtonsoft.json/index.json", r => exactMatchGetMetadataResult);
+            mockServer.Start();
+
+            // Redirect console output
+            using var consoleOutput = new StringWriter();
+            Console.SetOut(consoleOutput);
+
+            // Act
+            await PackageSearchRunner.RunAsync(
+                sourceProvider,
+                new List<string> { $"{mockServer.Uri}v3/index.json" },
+                "Fake.Newtonsoft.Json",
+                0,
+                20,
+                false,
+                true,
+                Common.NullLogger.Instance,
+                System.Threading.CancellationToken.None);
+
+            //stop mock server
+            mockServer.Stop();
+
+            // Assert
+            Assert.Equal(NormalizeNewlines(_onePackageExpectedOutput), NormalizeNewlines(consoleOutput.ToString()));
```

Comment:
My opinion for asserts is they should be as close to direct values as possible. Can the normalizing calls be done in new variables above, then have the assert just be for those 2 variables?